api/src/main/java/com/debugmafia/clueless/model/BoardLocation.java

```diff
@@ -0,0 +1,69 @@
+package com.debugmafia.clueless.model;
+
+import java.util.Set;
+
+public class BoardLocation {
+  private int xCoord;
+  private int yCoord;
```

Comment:
See comments regarding the updates to this class based off of the Professors feedback